Tasks/AzureDevOpsTestPlanV0/automatedTests.ts

```diff
@@ -0,0 +1,66 @@
+import tl = require('azure-pipelines-task-lib/task');
+import { testInvoker } from './automatedTestInvoker'
+import { TestPlanData, getAutomatedTestData } from './getAutomatedTests';
+import { TestCase } from 'azure-devops-node-api/interfaces/TestPlanInterfaces';
+
+
+export async function automatedTestsFlow(testSelectorInput: string) {
+
+    let ListOfTestsToBeExecuted: string[] = [];
+
+    console.log(tl.loc('automatedTestsTriggered'));
+    await getFQNsOfAutomatedTestCases()
+        .then((testsToBeExecuted) => {
+            ListOfTestsToBeExecuted = testsToBeExecuted;
+        })
+        .catch((error) => {
+            console.error("Error while fetching FqnsOfAutomatedTestCases :", error);
+        });
+
+    tl.debug("Invoking test execution for tests: " + ListOfTestsToBeExecuted);
+
+    if (ListOfTestsToBeExecuted !== null && ListOfTestsToBeExecuted !== undefined && ListOfTestsToBeExecuted.length > 0) {
+        testInvoker(ListOfTestsToBeExecuted);
+    }
+    else {
+        console.log("No automated tests found for given test plan inputs ");
+        if (testSelectorInput === 'manualTests') {
```

Comment:
Is this a single string variable or can it be a list containing both 'automatedTests' and 'manualTests' since we are allowing both at a time?

Reply:
It can have both, this check will never hit since this is automated flow. Let me invert the check.

if only manual= no issue
if only automated= fail if no automated tests found (inline with vstest)
if both = pass the task as manual tests got triggered

Reply:
updated, pls check